pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/invertedindex/TextIndexHandler.java

```diff
@@ -111,35 +117,22 @@ public void createTextIndexesOnSegmentLoad()
   }
 
   /**
-   * Right now the text index is supported on RAW (non-dictionary encoded)
+   * Right now the text index is supported on RAW and dictionary encoded
```

Comment:
same here. capture the constraints correctly in comments

Reply:
done